workflows/cluster_plots/__main__.py

```diff
@@ -0,0 +1,73 @@
+import json
+import os
+from pathlib import Path
+
+import luigi
+from google.cloud import storage
+
+from birdclef.knn_labels import (
+    compute_embedding_2d,
+    get_knn_labels,
+    get_label_agreement,
+    get_subset_pdf,
+    plot_distances,
+    plot_embedding,
+    write_plots_to_disk,
+)
+from birdclef.utils import get_spark
+from workflows.utils.pull import Pull
+from workflows.utils.push import Push
+
+from .plotting import ClusterPlotAllTasks
+
+
+class ClusterPlotTaskWrapper(luigi.WrapperTask):
+    data_input_path = luigi.Parameter()
+    data_output_path = luigi.Parameter()
+    plot_local_path = luigi.Parameter()
+    plot_cloud_path = luigi.Parameter()
+    list_species = luigi.Parameter()
+
+    def requires(self):
+        pull_data = Pull(
+            input_path=self.data_input_path,
+            output_path=self.data_output_path,
+            parallelism=os.cpu_count(),
+        )
+        yield pull_data
+
+        yield ClusterPlotAllTasks(
+            local_path=self.plot_local_path,
+            total_cnt=len(self.list_species),
+        )
+
+        for i in range(24):
+            name = self.list_species[i]
+            push_data = Push(
+                input_path=f"{self.plot_local_path}/{name}",
+                output_path=f"{self.plot_cloud_path}/{name}",
+                parallelism=1,
+            )
+            yield push_data
+
+
+def list_species(path):
+    return json.loads(Path(path).read_text())
+
+
+if __name__ == "__main__":
+    agreement_path = "/home/nzhon/data/processed/birdclef-2022/birdnet-embeddings-with-neighbors-agreement-static/v1/agreement.json"
```

Comment:
Make this relative to the project root, you should be calling the module at the root via `python -m workflows.cluster_plots`